dada2_chimera_taxa.R

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env Rscript
+
+# Read in package to read in command-line options.
+library("optparse")
+
+version <- "1.0"
+
+option_list <- list(
+
+  make_option(c("-i", "--input"), type="character", default=NULL,
+              help="Location of input .RDS file (required)." , metavar="path"),
+  
+  make_option(c("-r", "--refFasta"), type="character", default=NULL,
+              help="Location of reference fasta to use for taxa assignment (required).", 
+              metavar="path"),
+  
+  make_option(c("-s", "--ref_species"), type="character", default=NULL,
+              help="Location of reference fasta to use for species assignment (required).", 
+              metavar="path"),
+  
+  make_option(c("--seed"), type="integer", default=NULL,
+              help="Random seed to make command reproducible (required).", metavar = "integer"),
+  
+  make_option(c("-t", "--threads"), type="integer", default=1,
+              help="Number of threads to use (default: 1).", metavar="integer"),
+  
+  make_option(c("--skip_species"), action = "store_true", type="logical", default=FALSE,
```

Comment:
I think this is the correct default, but does add species remove some variants? I thought I remember reading that in one of the tutorials?

Reply:
Yep it does if there are inconsistencies between the genus and species assignments. I can change it so that 2 different tables are output.

Reply:
At this point it is hard to say which will be the most useful (species or the full table to genus level). Probably best to output both.